unit_tests/tests/test_launch.cpp

```diff
@@ -131,3 +131,64 @@ TEST(LaunchControl, CombinedCondition) {
 	EXPECT_FALSE(dut.isLaunchConditionMet(3200));
 
 }
+
+TEST(LaunchControl, CompleteRun) {
+	bool spark, fuel;
+	WITH_ENGINE_TEST_HELPER(TEST_ENGINE);
+
+	extern LaunchControlBase lc;
```

Comment:
uhhhh

Reply:
do you have a better way to do this? i could not find one.

Reply:
if you make `applyLaunchControlLimiting` a member function of `LaunchControlBase`, then you can construct an instance and call it directly without having to set up everything else and use `extern`

Reply:
i did it with inject in the init function, is that acceptable?